### PR TITLE
Make it obvious that host and port are both optional

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,11 +79,14 @@ Type 'help' for available commands/options.
...
-If the `-http="host:port"` option is specified, pprof starts a web server at
+If the `-http` flag is specified, pprof starts a web server at
 the specified host:port that provides an interactive web-based interface to pprof.
+Host is optional, and is "localhost" by default. Port is optional, and is a
+random available port by default. `-http=":"` starts a server locally at
+a random port.
 
 ```
-pprof -http=[host:port] [main_binary] profile.pb.gz
+pprof -http=<hostport> [main_binary] profile.pb.gz
```
Comment (aalexand) [inline README.md]: Maybe `pprof -http=[host]:[port] [main_binary] profile.pb.gz` would be cleaner?

File: doc/pprof.md [hunk 2]
```diff
@@ -54,12 +54,12 @@ commands.  Type `help` to get online help.
 
 ## Web interface
 
-If a port is specified on the command line:
+If a host:port is specified on the command line:
 
-    pprof -http=<port> [options] source
+    pprof -http=<hostport> [options] source
```
Comment (aalexand) [inline doc/pprof.md]: Ditto

File: internal/driver/cli.go [hunk 3]
```diff
@@ -292,7 +292,9 @@ var usageMsgSrc = "\n\n" +
 
 var usageMsgVars = "\n\n" +
 	"  Misc options:\n" +
-	"   -http host:port        Provide web based interface at host:port\n" +
+	"   -http host:port        Provide web based interface at host:port.\n" +
```
Comment (aalexand) [inline internal/driver/cli.go]: And here.

**Comment by rakyll** [comment]
@aalexand, PTAL.

**Pull request merged**